sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -361,16 +363,31 @@ def add_runner_options(parser):
                           state_stream, cleanup_callbacks)
 
 
-class PortableMetrics(metrics.metric.MetricResults):
+class PortableMetrics(metric.MetricResults,
+                      portable_metrics.ParseMonitoringInfoMixin):
```

Comment:
Please remove the use of double inheritance. You can use the methods from portable_metrics.ParseMonitoringInfoMixin by  making that file define the methods in the module instead of a class, since it only defined static methods